inst/chunks/bernoulli_likelihoods.stan

```diff
@@ -72,3 +72,56 @@
     else reject("Invalid link");
     return ll;
   }
+
+  /** 
+   * Log-normalizing constant in the clogit case
+   *
+   * @param N_j Integer number of observations in the j-th group
+   * @param D_j Integer number of successes in the j-th group
+   * @param eta_j Vector of linear predictions in the j-th group
+   * @return A scalar that normalizes the probabilities on the log-scale
+   */
+  real log_clogit_denom(int N_j, int D_j, vector eta_j);
```

Comment:
Looks like a duplicate line that was maybe originally planning on being used for external c++ code. I think it can be removed though, right?

Reply:
The `log_clogit_denom` function is recursive.